modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -877,13 +876,13 @@ private void _assertContainsBinary(
 		Pattern pattern = Pattern.compile(
 			String.format("/%s/%d/\\d+\\.\\d+$", className, classPK));
 
-		Stream<String> entriesStream = entries.stream();
+		entries = ListUtil.filter(entries, pattern.asPredicate());
```

Comment:
@JanisZhang This looks nice in tests. Could you check if you can apply the pattern to https://github.com/liferay-core-infra/liferay-portal/pull/3662 also?